app/models/theme_field.rb

```diff
@@ -4,6 +4,17 @@ class ThemeField < ActiveRecord::Base
 
   belongs_to :upload
 
+  scope :find_by_theme_ids, ->(theme_ids) {
+    return none unless theme_ids.present?
```

Comment:
Hmm `none` seems to be undefined.

Reply:
`none` is an ActiveRecord method like `all` etc. It returns an empty ActiveRecord relation (it doesn't query the DB).

https://apidock.com/rails/v4.0.2/ActiveRecord/QueryMethods/none

Reply:
Ahh icic. :+1: